File: frontend/src/dtos/Libro/create/LibroCreateRequestDto.ts
import { Genero } from '../../../enums/Genero'
export default class LibroCreateRequestDto
{
	nombre : string='';
	precio : number=0;
	genero : Genero = Genero.Poesia;
	autorId : number = 0;

	setNombre : (nombre : string) => LibroCreateRequestDto = (nombre: string) => {
		this.nombre = nombre;
		return this;
	}

	setPrecio : (precio : number) => LibroCreateRequestDto = (precio: number) => {
		this.precio = precio;
		return this;
	}
setGenero : (genero : Genero) => LibroCreateRequestDto = (genero: Genero) => {
		this.genero = genero;
		return this;
	}
	setAutorId : (autorId : number ) => LibroCreateRequestDto = (autorId: number) => {
		this.autorId = autorId;
		return this;
	}}

File: backend/src/main/java/com/example/template/dtos/LibroCreateRequestDTO.java
package com.example.template.dtos;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.util.*;
import com.example.template.enums.*;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class LibroCreateRequestDTO {
private String Nombre;
private double Precio;
private Genero Genero;
private Long AutorId;
}
